CHANGES_NEXT_RELEASE

```diff
@@ -2,4 +2,5 @@ Fix: modified engine version >=6 in packages.json
 Fix: update logops dependency from 1.0.8 to 2.1.0
 Fix: user provider timestamp correctly propagated to mapped entities when multientity pluging is used (#748)
 Add: new functions to set callbacks for removal of devices and groups (#735)
-Fix: add ?type parameter in CB request updates to avoid potential entity ambiguities (#733)
\ No newline at end of file
+Fix: add ?type parameter in CB request updates to avoid potential entity ambiguities (#733)
+Fix: incomplete HTTPS support for NGSI subscriptions (#593)
```

Comment:
Original PR (#594) used "NGSIv1" instead of "NGSI". Does the change means that at the end this PR is covering subscriptions both in NGSIv1 and NGSIv2?

(Looking to parent issue https://github.com/telefonicaid/iotagent-node-lib/issues/593#issuecomment-379726014 is seems that only NGSIv1 is covered, but maybe that comment is wrong)

Reply:
As you can see in the new test I added with this PR https://github.com/dcalvoalonso/iotagent-node-lib/blob/bug/uncomplete-https-support-subscriptions/test/unit/ngsiv2/general/https-support-test.js#268. registrations are already implemented in NGSIv2.

Probably you have also noticed that this PR only adds some unit tests. The changes the original PR(#594) had for https://github.com/telefonicaid/iotagent-node-lib/blob/master/lib/services/ngsi/subscriptionService.js were integrated by @AlvaroVega in ca8f85036f2f5577b2903f784da5f3f659688340 and covered also NGSIv2.

Reply:
Thanks for the clarification!

NTC